### PR TITLE
Pass path to configuration file to the tile processor.

### PR DESCRIPTION
**Pull request by zerebubuth**
Connects to mapzen/tilequeue#70.

@rmarianski could you review, please?

### CODE REVIEW

**Comment by zerebubuth** [comment]
Refs [comment](https://github.com/mapzen/tilequeue/pull/77#issuecomment-191219469) describing these changes in more detail.

File: tileserver/__init__.py [hunk 1]
```diff
@@ -171,7 +172,7 @@ class TileServer(object):
 
     def __init__(self, layer_config, data_fetcher, post_process_data,
                  io_pool, store, redis_cache_index, sqs_queue,
-                 health_checker=None):
+                 config_file_path, health_checker=None):
```
Comment (rmarianski) [inline tileserver/__init__.py]: Don't need this here anymore, right?

File: tileserver/__init__.py [hunk 2]
```diff
@@ -180,6 +181,8 @@ def __init__(self, layer_config, data_fetcher, post_process_data,
         self.redis_cache_index = redis_cache_index
         self.sqs_queue = sqs_queue
         self.health_checker = health_checker
+        self.config_file_path = config_file_path
+        self.cache = dict()
```
Comment (rmarianski) [inline tileserver/__init__.py]: Doesn't seem like we need this cache object here any more.

File: tileserver/__init__.py (hunk 1 above)
Comment (zerebubuth) [inline tileserver/__init__.py]: Well spotted! Removed in acb4b2d.

File: tileserver/__init__.py (hunk 2 above)
Comment (zerebubuth) [inline tileserver/__init__.py]: Nope, I missed that, sorry. Removed in acb4b2d.

**Comment by rmarianski** [comment]
:+1:

**Pull request merged**